src/main/kotlin/no/nav/helse/fritakagp/web/api/AltinnRettigheterRoute.kt

```diff
@@ -31,4 +40,22 @@ fun Route.altinnRoutes(authRepo: AltinnClient) {
             }
         }
     }
+    route("/arbeidsgiver-tilganger") {
+        get("/") {
```

Comment:
Ikke helt stødig på notasjonen her, men har en følelse av at denne bør kunne sløyfes?
```suggestion
        get {
```